tests/basic_service_test.go

```diff
@@ -159,18 +159,23 @@ func TestBasicService(t *testing.T) { // nolint
 				fromToPeer[p.Name] = endpoint
 			}
 
-			// Same client reach with different port to the session affinity service, should have same destination pod,
-			zap.L().Info(fmt.Sprintf("Testing connections to different ports of sesson affinity service, should use same from/to peers: %v", fromToPeer))
+			// to validate if affiliation applies to same port
+			zap.L().Info(fmt.Sprintf("Testing connections to same ports, try multiple times to affirm the affiliation, should use same from/to peers: %v", fromToPeer))
```

Comment:
can we split this in different tests? 
the title is saying we have different ports but we change the behavior in the middle of it.